src/tests.builds

```diff
@@ -13,7 +13,10 @@
   <Import Project="$(ToolsDir)VersionTools.targets" Condition="Exists('$(ToolsDir)VersionTools.targets')" />
 
   <ItemGroup>
-    <Project Include="$(MSBuildThisFileDirectory)*\tests\**\*.Tests.csproj" Exclude="@(TestProjectExclusions)">
+    <Project Condition="'$(Performance)'=='true'" Include="$(MSBuildThisFileDirectory)System.Collections*\tests\**\*Performance.Tests.csproj" Exclude="@(TestProjectExclusions)">
```

Comment:
Is this temporary as I don't want to do filtering for performance in the test.builds file itself.